specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-07-01/examples/Remediations_CancelResourceGroupScope.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "subscriptionId": "35ee058e-5fa0-414c-8145-3ebb8d09b6e2",
+    "resourceGroupName": "rg1",
```

Comment:
Can you change it to `myResourceGroup` for resourceGroupName example values, which is the conventional name used in examples.